tasks/sep/006-0-sc-guardian-updates/VALIDATION.md

```diff
@@ -0,0 +1,74 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Overrides
+
+The following state overrides should be seen:
+
+### `0xa87675ebb9501C7baE8570a431c108C1577478Fa` (The Security Council Safe)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0xE75Cd021F520B160BF6b54D472Fa15e52aFe5aDD)
+
+Enables the simulation by setting the threshold to 1:
+
+- **Key:** `0x0000000000000000000000000000000000000000000000000000000000000004` <br/>
+  **Value:** `0x0000000000000000000000000000000000000000000000000000000000000001`
+
+## State Changes
+
+### `0x54e8baccc67fa3c6b3e9a94baa4d70d1668f0820` (`LivenessGuard`)
+
+Links:
+- [Etherscan](https://sepolia.etherscan.io/address/0x54e8baccc67fa3c6b3e9a94baa4d70d1668f0820) (not verified)
+
+State Changes:
+- **Key:** `0x023c107741c8297b7d31ce48158a7d3e0c1bab34eb099ba05b48e6e0bb5b0324` <br/>
+  **Before:** `0x000000000000000000000000000000000000000000000000000000006620aaa4` <br/>
+  **After:** `0x000000000000000000000000000000000000000000000000000000006621dcfe` <br/>
+  **Meaning:** The 'lastLive' timestamp of the caller has been updated. The key can be verified by:
+    1. Seeing that the [slot of the mapping is 0](https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/snapshots/storageLayout/LivenessGuard.json#L6).
+    2. obtaining the storage key with `cast`, where the address provided should be the one used to simulate.
+      ```shell
+        cast index address 0x2e2e33fedd27fdecfc851ae98e45a5ecb76904fe 0
```

Comment:
Where is `0x2e2e33fedd27fdecfc851ae98e45a5ecb76904fe` from?

Reply:
It all looks good, just want to understand why this address is used.

Reply:
Good Q! This is the first address returned by `safe.getOwners()` when using `SIMULATE_WITHOUT_LEDGER=1` ([ref](https://github.com/ethereum-optimism/superchain-ops/blob/maur/sep-sc-guardian-change/single.just#L22)). 

You should see it printed in the output of `just simulate`: 

![2024-04-19 at 10 18@2x](https://github.com/ethereum-optimism/superchain-ops/assets/23033765/566045d2-1ea7-4e8f-a48d-3cbe831ead3b)